### PR TITLE
 Refactor CI workflows

### PR DESCRIPTION
**Pull request by hkusu**
## Issue
- N/A

## Overview (Required)
- Refactor CI workflows.
- In my forked repository, the modified workflow is successful.
  - https://github.com/hkusu/conference-app-2021/actions/runs/653045758
  - https://github.com/hkusu/conference-app-2021/actions/runs/653045757

## Links
-  

## Screenshot

Screen not changed.

### CODE REVIEW

File: .github/workflows/assemble-and-stage.yml [hunk 1]
```diff
@@ -33,11 +33,6 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ steps.fetch-head.outputs.ref }}
-      - run: rm -fr ./gh-actions
-      - uses: actions/checkout@v2
-        with:
-          ref: gh-actions
-          path: gh-actions
```
Comment (hkusu) [inline .github/workflows/assemble-and-stage.yml:40]: Not used in this job

File: .github/workflows/assemble-and-stage.yml [hunk 2]
```diff
@@ -74,11 +69,6 @@ jobs:
...
```
Comment (hkusu) [inline .github/workflows/assemble-and-stage.yml:81]: Not used in this job

File: .github/workflows/test-ios.yml [hunk 3]
```diff
@@ -26,11 +26,6 @@ jobs:
       - uses: actions/checkout@v2
         with:
           ref: ${{ steps.fetch-head.outputs.ref }}
-      - run: rm -fr ./gh-actions
-      - uses: actions/checkout@v2
-        with:
-          ref: gh-actions
-          path: gh-actions
```
Comment (hkusu) [inline .github/workflows/test-ios.yml:33]: Not used in this job

File: .github/workflows/assemble-and-stage.yml [hunk 4]
```diff
@@ -92,9 +82,12 @@ jobs:
             **/gradle.properties
             gradle/**
       - uses: actions/upload-artifact@v2
+        if: cancelled() != true
```
Comment (hkusu) [inline .github/workflows/assemble-and-stage.yml:85]: Upload even if there is an error in the test

File: .github/workflows/test-ios.yml [hunk 5]
```diff
@@ -48,8 +43,10 @@ jobs:
             **/gradle.properties
             gradle/**
       - uses: actions/upload-artifact@v2
+        if: cancelled() != true
```
Comment (hkusu) [inline .github/workflows/test-ios.yml:46]: Upload even if there is an error in the test

File: .github/workflows/test-ios.yml [hunk 6]
```diff
@@ -48,8 +43,10 @@ jobs:
...
         with:
           name: unit-test-ios-reports
-          path: "./**/reports"
+          path: |
+            **/reports
+            !buildSrc/build/reports
```
Comment (hkusu) [inline .github/workflows/test-ios.yml:51]: Exclude unnecessary files

File: .github/workflows/assemble-and-stage.yml [hunk 7]
```diff
@@ -92,9 +82,12 @@ jobs:
...
         with:
           name: unit-test-reports
-          path: "./**/reports"
+          path: |
+            **/reports
+            !buildSrc/build/reports
```
Comment (hkusu) [inline .github/workflows/assemble-and-stage.yml:90]: Exclude unnecessary files

**Comment by takahirom** [comment]
👀

**Review by takahirom — APPROVED** [review]
LGTM 👍

**Comment by takahirom** [comment]
🆒

**Pull request merged**